docs/source/filters.rst

```diff
@@ -5,21 +5,19 @@
 Filters
 =======
 
-Filters in Ansible are from Jinja2, and are used for transforming data inside
-a template expression. Jinja2 ships with many filters as does Ansible and also
-allows users to add their own custom filters.
+Filters in Ansible are from Jinja2, and are used to transform data inside
+a template expression. The templates operate on the Ansible controller, and not
+on the target host. Therefore, filters execute on the controller as they augment
+the data locally.
 
-It should be noted that templates operate on the Ansible controller, not on the
-target host, so filters execute on the controller as they augment the data
-locally.
+Jinja2 ships with many filters as does Ansible, and also allows users to add
+their own custom filters.
```

Comment:
do you intend to talk about similar capabilities of ansible and jinja2? as in they both let users add custom filters? I understood it as that and just reorganized the sentences a little to keep the flow intact.

Reply:
What you wrote reads fine to me. Thank you.